### PR TITLE
Fix missing Request parameter for Adgeneration Adapter

### PR DESCRIPTION
**Pull request by AdGeneration**
## Type of change
fix: add UA to RequestHeader.
fix: fix sizes query.

　　　　　　　　　　　
## Description of change
Change currency of ad-generation's bidResponse according to bidRequest

Maintainer: ssp-ope@supership.jp

### CODE REVIEW

File: adapters/adgeneration/adgeneration.go [hunk 1]
```diff
@@ -54,6 +54,9 @@ func (adg *AdgenerationAdapter) MakeRequests(request *openrtb.BidRequest, reqInf
 	headers := http.Header{}
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
+	if request.Device != nil && len(request.Device.UA) > 0 {
+		headers.Add("User-Agent", request.Device.UA)
```
Comment (SyntaxNode) [inline adapters/adgeneration/adgeneration.go:58]: Please add unit test coverage for this new branch.

File: adapters/adgeneration/adgeneration.go [hunk 2]
```diff
@@ -106,11 +109,14 @@ func (adg *AdgenerationAdapter) getRawQuery(id string, request *openrtb.BidReque
...
 	adSize := getSizes(imp)
 	if adSize != "" {
-		v.Set("size", adSize)
+		v.Set("sizes", adSize)
 	}
 	if request.Site != nil && request.Site.Page != "" {
 		v.Set("tp", request.Site.Page)
 	}
+	if request.Source != nil && request.Source.TID != "" {
+		v.Set("transactionid", request.Source.TID)
```
Comment (SyntaxNode) [inline adapters/adgeneration/adgeneration.go:118]: Please add unit test coverage for this new branch.

**Comment by AdGeneration** [comment]
Added test items for the added parameters.
Also, there was a problem with some test code, so I have fixed it as well.

**Review by SyntaxNode — APPROVED** [review]

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**